docs/presto-admin-commands.rst

```diff
@@ -411,6 +416,10 @@ Example
 ::
 
     sudo ./presto-admin server install /tmp/presto.rpm
+    sudo ./presto-admin server install 0.144.6
```

Comment:
It doesn't right now. That will have to be added later when they can be automatically uploaded and are accessible. I think there's some issues right now because they have some additional features that shouldn't be publicly available. Matt said he would address this after this functionality is added.